ambari-server/pom.xml

```diff
@@ -12,15 +12,14 @@
   <parent>
     <groupId>org.apache.ambari</groupId>
     <artifactId>ambari-project</artifactId>
-    <version>2.0.0.0-SNAPSHOT</version>
+    <version>${revision}</version>
```

Comment:
Around line 420 of this file, we enforce Maven 3.3.9 in `maven-enforcer-plugin` - should that also be updated in this commit since we're using `revision` ?

Reply:
Yes, we should - I'll remote it from here and have it defined in the parent.

Reply:
Nice catch.  I guess that usage can now be removed.

https://github.com/apache/ambari/blob/246e9603a319b21e23f5e5a0f757b78b351ea546/ambari-server/pom.xml#L407-L426

Reply:
I've removed this in favor of the top-level parent's definition of the enforcer plugin.